rsl_tools.py

```diff
@@ -0,0 +1,78 @@
+import requests
```

Comment:
As you mentioned elsewhere, you might consider adding the following to the top of the file

```
import sys
import subprocess

if not 'requests' in sys.modules:
    subprocess.check_call([sys.executable, '-m', 'pip', 'install', 'requests'])
```

Reply:
Though it's possible that someone is using a different package manager than pip.... But then they probably know how to install 'requests' in that case.